apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -64,9 +78,29 @@ const readOrganizations = async ({ context, meters }) => {
         where: {
             id_in: organizationIds,
         },
+        deletedAt: null,
+    })
+}
+
+const readMeterReportingPeriods = async ({ context, organizations }) => {
+    const organizationIds = organizations.map(org => org.id)
+    return await loadListByChunks({
+        context,
+        list: MeterReportingPeriod,
+        where: {
+            organization: {
+                id_in: organizationIds,
+            },
+            deletedAt: null,
+        },
     })
 }
 
+const checkIsDateInPeriod = (date, today, start, end) => {
```

Comment:
This function must be tested well. Also, this is truly hard to read this peace of code.
Why we have to pass today date to the function?
What if start is 1 and end is 10? Will 3th day of months to be in the period?

Reply:
Perhaps it should be called in an arc. In fact, it just checks that the current date coincides with the beginning or end of the period, since pushes are sent only on these days, the rest is skipped